### DISCUSSION TITLE
How to start and stop multiple schedules

### DISCUSSION THREAD

**Discussion by raiAmagi** [dagster-io/dagster#22490] [Q&A] [answered]
Please let me know the following items in the commands to start and stop the schedule.

Command: dagster schedule [start|stop]

1. how to specify multiple schedules when starting/stopping
　　I tried the following method, but it resulted in an error.
　　　Help shows [SCHEDULE_NAME]... I think it is possible to specify more than one schedule since it is displayed as [SCHEDULE_NAME]....
　　　　1) dagster schedule stop schedule01 schedule02
　　　　2) dagster schedule stop schedule01,schedule02
　　　　3) dagster schedule stop [schedule01,schedule02]...

2. how to specify all schedules when stopping
　　The --start-all option is available for start, but there is no equivalent option for stop.

**Comment by garethbrickman** [comment, marked as answer]
The CLI does not support doing either of these out of the box, but you could create a shell script to approximate them. Example: 

```bash
#!/bin/bash

# List of schedules to start/stop
SCHEDULES=("schedule01" "schedule02" "schedule03")

# Operation to perform: start or stop
OPERATION=$1

# Check if the operation is valid
if [[ "$OPERATION" != "start" && "$OPERATION" != "stop" ]]; then
  echo "Invalid operation. Use 'start' or 'stop'."
  exit 1
fi

# Iterate over the list of schedules and perform the operation
for SCHEDULE in "${SCHEDULES[@]}"; do
  echo "Attempting to $OPERATION schedule: $SCHEDULE"
  dagster schedule $OPERATION $SCHEDULE
  if [[ $? -ne 0 ]]; then
    echo "Failed to $OPERATION schedule: $SCHEDULE"
  else
    echo "Successfully $OPERATIONed schedule: $SCHEDULE"
  fi
done
```

`./manage_schedules.sh start`

**Comment by raiAmagi** [comment]
Thanks for the reply.

Thank you for presenting the status of support for multiple schedule name designations and another idea using scripts.

I will refer to it.
